comps/comps-foreman-plugins-fedora21.xml

```diff
@@ -110,6 +110,7 @@
       <packagereq type="default">rubygem-foreman_graphite-doc</packagereq>
       <packagereq type="default">rubygem-foreman_hooks-doc</packagereq>
       <packagereq type="default">rubygem-foreman_host_rundeck-doc</packagereq>
+      <packagereq type="default">rubygem-foreman_chef-doc</packagereq>
```

Comment:
ouch, I ran comps_doc.sh but it recognized my locale and considered 'ch' belongs after 'h', you know we have 'ch' as a separate letter... I'll try to memorize to set LC_ALL=C next time